src/System.Runtime.Loader/ref/System.Runtime.Loader.csproj

```diff
@@ -4,8 +4,10 @@
   <PropertyGroup>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="System.Reflection.Metadata.cs" />
```

Comment:
Thanks. I should I guess call out why we do this which is we have tools that we use to generate these and they generate the entire reference assembly in one cs file.